src/test/scala/mesosphere/marathon/api/v2/PodsResourceTest.scala

```diff
@@ -4,116 +4,191 @@ import javax.servlet.http.HttpServletResponse
+        val parsedResponse = Option(response.getEntity.asInstanceOf[String]).map(Json.parse)
+        parsedResponse should not be (None)
+        parsedResponse.map(_.as[Pod]) should not be (None) // validate that we DID get back a pod definition
 
-      response.getMetadata().containsKey(PodsResource.DeploymentHeader) should be(true)
+        response.getMetadata.containsKey(PodsResource.DeploymentHeader) should be(true)
+      }
     }
-  }
 
-  test("delete a pod") {
-    implicit val podSystem = mock[PodManager]
-    val f = Fixture.create()
+    "delete a pod" in {
+      implicit val podSystem = mock[PodManager]
+      val f = Fixture()
 
-    podSystem.find(any).returns(Future.successful(Some(PodDefinition())))
-    podSystem.delete(any, eq(false)).returns(Future.successful(DeploymentPlan.empty))
-    val response = f.podsResource.remove("/mypod", false, f.auth.request)
+      podSystem.find(any).returns(Future.successful(Some(PodDefinition())))
+      podSystem.delete(any, eq(false)).returns(Future.successful(DeploymentPlan.empty))
+      val response = f.podsResource.remove("/mypod", false, f.auth.request)
 
-    withClue(s"response body: ${response.getEntity}") {
-      response.getStatus() should be(HttpServletResponse.SC_ACCEPTED)
+      withClue(s"response body: ${response.getEntity}") {
+        response.getStatus should be(HttpServletResponse.SC_ACCEPTED)
 
-      val body = Option(response.getEntity.asInstanceOf[String])
-      body should be(None)
+        val body = Option(response.getEntity.asInstanceOf[String])
+        body should be(None)
 
-      response.getMetadata().containsKey(PodsResource.DeploymentHeader) should be(true)
+        response.getMetadata.containsKey(PodsResource.DeploymentHeader) should be(true)
+      }
     }
-  }
 
-  test("lookup a specific pod, and that pod does not exist") {
-    implicit val podSystem = mock[PodManager]
-    val f = Fixture.create()
+    "lookup a specific pod, and that pod does not exist" in {
+      implicit val podSystem = mock[PodManager]
+      val f = Fixture()
 
-    podSystem.find(any).returns(Future.successful(Option.empty[PodDefinition]))
-    val response = f.podsResource.find("/mypod", f.auth.request)
+      podSystem.find(any).returns(Future.successful(Option.empty[PodDefinition]))
+      val response = f.podsResource.find("/mypod", f.auth.request)
 
-    withClue(s"response body: ${response.getEntity}") {
-      response.getStatus() should be(HttpServletResponse.SC_NOT_FOUND)
-      val body = Option(response.getEntity.asInstanceOf[String])
-      body should not be (None)
-      body.foreach(_ should include("mypod does not exist"))
+      withClue(s"response body: ${response.getEntity}") {
+        response.getStatus should be(HttpServletResponse.SC_NOT_FOUND)
+        val body = Option(response.getEntity.asInstanceOf[String])
+        body should not be (None)
+        body.foreach(_ should include("mypod does not exist"))
+      }
+    }
+
```

Comment:
all changes in ```PodsResourceTest``` before this line was just changing the test format style, or did I miss a semantically change?